guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>This sentence states that <em>Here are examples for working with Initiatives and Events using </em><a href="https://github.com/Esri/arcgis-python-api/raw/d67a9543e2b29a82f2cd1cb10d186bf7cbdfcae8/guide/guide/hub-for-guide-premium" target="_blank"><em>ArcGIS Hub Premium</em></a><em>, site and page layouts, and theme editing using </em><a href="https://github.com/Esri/arcgis-python-api/raw/d67a9543e2b29a82f2cd1cb10d186bf7cbdfcae8/guide/guide/enterprise-sites" target="_blank"><em>Enterprise Sites</em></a><em>.</em> but the first sentence states we're connecting to a hub with Hub Basic capabilities. This sentence should read <em>Here are examples for working with Sites and Pages using ArcGIS Hub Basic.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>